File: solutions/csharp/high-scores/2/HighScores.cs
using System;
using System.Collections.Generic;
using System.Linq;

public class HighScores
{
    private List<int> _scores = null;
    
    public HighScores(List<int> list)
    {
        _scores = list;
    }

    public List<int> Scores() => _scores;

    public int Latest() => _scores.Last();

    public int PersonalBest() => _scores.Max();

    public List<int> PersonalTopThree() =>
        _scores.OrderByDescending(score => score).Take(3).ToList();
}

File: solutions/python/high-scores/2/high_scores.py
class HighScores:
    def __init__(self, scores):
        self.scores = scores
    def latest(self):
        return self.scores[-1]
    def personal_best(self):
        sorted_scores = self.scores.copy()
        sorted_scores.sort()
        return(sorted_scores[-1])
    def personal_top_three(self):
        sorted_scores = self.scores.copy()
        sorted_scores.sort(reverse=True)
        return(sorted_scores[:3])
